### ISSUE TITLE
Logo on docs.raymon.ai broken

### ISSUE THREAD

**Issue by maartenaelvoet** [raymon-ai/raymon#79]
- logo broken
- alt text should be: Raymon.ai
![image](https://user-images.githubusercontent.com/2451526/125475904-077ca42e-1894-402c-9df9-9f584de88ecf.png)

**Comment by kvhooreb** [comment]
Fixed & hosting docs on github pages now
